### PR TITLE
Version bumping and tagging

### PR DESCRIPTION
**Pull request by glooms**

### CODE REVIEW

File: release/release.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  fi
+}
+
+case $1 in
+  "major"|"minor"|"patch")
+    sanity_check
+    bump_version $1
+    echo $VERSION
+    WD=$(pwd)
+    cd ../spec
```
Comment (wennmo) [inline release/release.sh]: This assumes that we are in the `/release` directory? Is that the case even if the script is run with e.g. an absolute path? You can use `cd "$(dirname "$0")"` to goto the script directory.

File: release/verval/verval_test.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
+package main
```
Comment (wennmo) [inline release/verval/verval_test.go]: Are these tests included in the CCI config?

Comment (glooms) [reply release/verval/verval_test.go]: I will check!

File: release/release.sh (hunk 1 above)
Comment (glooms) [reply release/release.sh]: True! Will fixelifix!

Comment (glooms) [reply release/release.sh]: Fixed

File: release/verval/verval_test.go (hunk 2 above)
Comment (glooms) [reply release/verval/verval_test.go]: Added to cci

**Pull request merged**